app/helpers/application_helper.rb

```diff
@@ -11,6 +11,11 @@ def application_url
     Rails.configuration.application_url
   end
 
+  def page_title(page_title = '')
```

Comment:
I don't like the parameter name same as the method name, so maybe:

```suggestion
  def page_title(title = '')
```

Reply:
Renamed with https://github.com/kaigionrails/conference-app/pull/256/commits/2f81f47d6870139d3fd6081c376285d9d54748a6